manifests/k/Koenvh/astiga/0.1.2/Koenvh.astiga.locale.en-US.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Koenvh.astiga
+PackageVersion: 0.1.2
+PackageLocale: en-US
+Publisher: Koen
+PublisherUrl: https://asti.ga
+PackageName: Astiga

```

Comment:
```suggestion
PackageName: astiga
```
to match what's shown in Control Panel:
![image](https://user-images.githubusercontent.com/11600822/127692193-c4e37011-54d1-4f9b-ba26-16fbe304249c.png)